### ISSUE TITLE
Can you come up with a detailed README file, I don't really understand this docker configuration you have now

### ISSUE THREAD

**Issue by Cyclones-Y** [submit-paper/Danzero_plus#7]
Forgive my ignorance.
I didn't quite understand the configuration of the program
1. Start two dockers in learnring and action respectively? and then populate action.py with the docker IP address for learning?
2. Why do you need to start so many ip addresses in your create_container.sh?
3. Maybe you're busy right now, but I'd appreciate it if you could come up with a detailed README file!

**Comment by submit-paper** [comment]
Well, as we adopt distributed reinforcement learning, we start actors and learner in different dockers. In this way, the actors only use the CPUs while the learner adopts the GPU for training. Each actor is in its own docker and equipped with its IP address for data transportation and model synchronization. In this way, many ip addresses are needed if you want to run multiple actors for higher efficiency. This is also due to that the gamecore we use is of low efficiency so we have to use multiple actors.

**Comment by Cyclones-Y** [comment]
> Well, as we adopt distributed reinforcement learning, we start actors and learner in different dockers. In this way, the actors only use the CPUs while the learner adopts the GPU for training. Each actor is in its own docker and equipped with its IP address for data transportation and model synchronization. In this way, many ip addresses are needed if you want to run multiple actors for higher efficiency. This is also due to that the gamecore we use is of low efficiency so we have to use multiple actors.


Thanks a lot for your answer, that means I only need to open different docker containers on one machine to deploy your project right?

**Comment by submit-paper** [comment]
Yes, but you have to use docker network to assign ip for each docker.

**Comment by Choysang** [comment]
> > 嗯，因为我们采用的是分布式强化学习，所以我们在不同的docker中启动actor和learner。这样actor只使用CPU，而learner使用GPU进行训练。每个actor都在自己的docker中，并配备自己的IP地址，用于数据传输和模型同步。这样，如果要运行多个actor以提高效率，就需要很多ip地址。这也是因为我们使用的游戏核心效率低，所以我们必须使用多个actor。
> 
> 非常感谢您的回答，这意味着我只需要在一台机器上打开不同的docker容器就可以部署您的项目，对吗？

Hello, I saw your response regarding Docker and wanted to ask if you could share the Docker files. The shared link has expired, and the author is no longer maintaining it. Thank you! Here is my contact information:choysunzz@163.com
